react/features/invite/components/info-dialog/InfoDialog.web.js

```diff
@@ -246,6 +257,20 @@ class InfoDialog extends Component<Props, State> {
                     value = { this._getTextToCopy() } />
             </div>
         );
+
+        if (isInlineDialog) {
+            return inlineDialog;
+        }
+
+        return (
+            <Dialog
+                cancelTitleKey = 'dialog.close'
+                submitDisabled = { true }
+                titleKey = 'info.label'
+                width = 'small'>
+                { inlineDialog }
+            </Dialog>
+        );
```

Comment:
It makes more sense to me if this component in general were called MeetingInfo and then the consumer of MeetingInfo could dictate how to display it, be it in a Dialog or on its own. Up to you.

Reply:
But then we'll need 2 more classes/files for adding the Dialog and the InlineDialog wrappers, isn't it?